main.cpp

```diff
@@ -0,0 +1,179 @@
+#if !DEVICE_QSPI
```

Comment:
license header file please on the top of the file

Reply:
done